src/engine/engine.cpp

```diff
@@ -44,6 +44,12 @@ const char *GetCampaignPath(void) {
     return cv_campaign_path->s_value;
 }
 
+const char *GetFullCampaignPath(void) {
```

Comment:
Suggest returning std::string to avoid need for static buffer

Reply:
Can't because we still have older C code, but it's a good point for later.